### PR TITLE
add basic functionality and styles for more menu

### LINKED ISSUE #2: Create a Publicly Accessible Demo Page
Create an accessible github page that shows a demo for the popover.

### PR DESCRIPTION
**Pull request by craigmichaelmartin**
@akersinformz
closes #2 
Hosted: https://cinbcuniversal.github.io/suprematism-more-menu/

### CODE REVIEW

**Comment by craigmichaelmartin** [comment]
This is ready to be reviewed. Hosted https://cinbcuniversal.github.io/suprematism-more-menu/

File: example/app/app.component.ts [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+        class="js-dynamicallyVividMiddleMenu">
+      <supre-more-menu-item supreText="First menu item"></supre-more-menu-item>
+      <supre-more-menu-item supreText="Second menu item which just seems to keep going and going"></supre-more-menu-item>
+      <supre-more-menu-item supreText="Third"></supre-more-menu-item>
+    </supre-more-menu>
+  `
+})
+export class ExampleDynamicallyVividMiddleAligned {
+  private supreVivid;
+  @HostListener('focusin') @HostListener('mouseenter') activateOnHover() { this.supreVivid = true; }
```
Comment (akersinformz) [inline example/app/app.component.ts]: Should we have the supre-more-menu component listen for these events versus the component implementing the supre-more-menu?

Comment (craigmichaelmartin) [inline example/app/app.component.ts]: This is demonstrating an example parent component passing extrinsic data to the more-menu component through a directive. I happened to have the data changing based on the parent's hover (could be anything) for demonstration purposes. Does that make sense / am correctly understanding your question?

File: out-tsc/more-menu.component.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
+:host {
+  display: inline-block; }
+
+.moreMenu-iconContainer {
```
Comment (craigmichaelmartin) [inline out-tsc/more-menu.component.css]: Good catch!

Comment (craigmichaelmartin) [inline out-tsc/more-menu.component.css]: This is resolved.

**Comment by craigmichaelmartin** [comment]
@akersinformz @caffeinelogic This is ready to re-review. I'm working on removing the directives for `supreAlign`, `supreVivid`, and `supreVisible` and instead having just one `supreOptions$`. I think it would be best reviewed on its own, though, and shouldn't hold up this code. If you'd rather wait, that's fine!

**Comment by akersinformz** [comment]
Spoke with Craig. He is going to move forward with streaming in the options for the more-menu.

**Comment by craigmichaelmartin** [comment]
Ready to review. This now uses one directive that accepts an options streams.

**Pull request merged**